shared/4-hydrogen-vm-render-script.js

```diff
@@ -147,12 +149,38 @@ async function mountHydrogen() {
   // page don't say `undefined`.
   urlRouter.attach();
 
+  //const storage = new Storage(null, window.indexedDB, null, false, localStorage, platform.logger);
+  const storageFactory = new StorageFactory(
+    null,
+    window.indexedDB,
+    window.FDBKeyRange,
+    window.localStorage
+  );
+  const storage = await storageFactory.create('1', platform.logger);
+
+  const hsApi = new HomeServerApi({
+    request: (url, options) => {
+      return {
+        abort() {},
+        async response() {
+          return {
+            status: 204,
+            body: {},
+          };
```

Comment:
Respond to the `/context` requests about missing event relations with empty content.

We don't want more errors to be logged but we're ok with the missing events staying missing.

Reply:
Added as a comment.